router_client.py

```
@@ -44,13 +80,36 @@ async def get_devices(self) -> list[dict[str, Any]] | None:
         
         try:
             async with self.session.get(
-                self.url, headers=headers, ssl=False, timeout=30
+                self.url, headers=headers, ssl=False, timeout=30, allow_redirects=False
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)